docs/UserGuide.adoc

```diff
@@ -196,26 +208,30 @@ View recipes of dishes served at the restaurant.
 
 Format: `listRecipes`
 
-For each recipe, the list of ingredients and its corresponding amount in units required for the standard serving will be displayed.
+****
+• For each recipe, the list of ingredients and its corresponding amount in units required for the standard serving will be displayed.
 Each recipe will be indexed.
+****
```

Comment:
In my opinion, ingredients should come before recipe- users are probably confused